### ISSUE TITLE
MSI Mortar Artic H270M Working

### ISSUE THREAD

**Issue by smithj** [nagisa/msi-rgb#34]
No extensive testing, but it worked 1st try...  Blue is dim for some reason, but I'm thinking it's my questionable LED strip.

**Comment by smithj** [comment]
Not exactly.. The one listed in the readme is the tomahawk artic.. this is the mortar. probably the same, but uATX

**Comment by nagisa** [comment]
Aha youre right, ill list ir later today.
